ledger-service/http-json/src/main/scala/com/digitalasset/http/EndpointsCompanion.scala

```diff
@@ -58,16 +73,15 @@ object EndpointsCompanion {
   object Error {
     implicit val ShowInstance: Show[Error] = Show shows {
       case InvalidUserInput(e) => s"Endpoints.InvalidUserInput: ${e: String}"
-      case ParticipantServerError(s, d) =>
-        s"Endpoints.ParticipantServerError: ${s: GrpcCode}${d.cata((": " + _), "")}"
```

Comment:
Is it maybe easier to just give `s` a better name like `grpcCode` and then do `${grpcCode}`?
Also would prefer to do the fold/cata in a line above it and assign to val, little easier to read